pkgs/applications/networking/sync/desync/default.nix

```diff
@@ -1,20 +1,21 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "desync";
-  version = "0.4.0";
+  version = "0.9.0";
   rev = "v${version}";
```

Comment:
This has no use here IMO. Could you put it as is in the arguments to `fetchFromGitHub`?

Reply:
Done, as of 4f8057ec837d0c9a79930c21f1673ebc98074d67